crates/fluvio-smartengine/src/transforms/simple_transform.rs

```diff
@@ -0,0 +1,83 @@
+use fluvio_smartmodule::dataplane::smartmodule::{
+    SmartModuleInput, SmartModuleOutput, SmartModuleTransformErrorStatus,
+};
+use wasmtime::{TypedFunc, AsContextMut};
+use anyhow::Result;
+
+use crate::{
+    instance::{SmartModuleInstanceContext, SmartModuleTransform},
+    state::WasmState,
+};
+
+type WasmFn = TypedFunc<(i32, i32, u32), i32>;
+
+pub(crate) const FILTER_FN_NAME: &str = "filter";
+pub(crate) const MAP_FN_NAME: &str = "map";
+pub(crate) const FILTER_MAP_FN_NAME: &str = "filter_map";
+pub(crate) const ARRAY_MAP_FN_NAME: &str = "array_map";
+
+pub(crate) struct SimpleTansform {
+    f: WasmFn,
+    name: String,
+}
+
+impl std::fmt::Debug for SimpleTansform {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.name.as_str() {
+            FILTER_FN_NAME => write!(f, "FilterFn"),
```

Comment:
no need to distinguish fn name. just write it out

Reply:
Just wanted to keep it the same as before :) Updated.